lib/options.go

```diff
@@ -186,80 +186,80 @@ type Options struct {
 
 	// Initial values for VUs, max VUs, duration cap, iteration cap, and stages.
 	// See the Runner or Executor interfaces for more information.
-	VUs        null.Int           `json:"vus" envconfig:"vus"`
-	VUsMax     null.Int           `json:"vusMax" envconfig:"vus_max"`
-	Duration   types.NullDuration `json:"duration" envconfig:"duration"`
-	Iterations null.Int           `json:"iterations" envconfig:"iterations"`
-	Stages     []Stage            `json:"stages" envconfig:"stages"`
+	VUs        null.Int           `json:"vus" envconfig:"vus" js:"vus"`
```

Comment:
What are the `js:` struct tags used for?

Reply:
[here](https://github.com/loadimpact/k6/blob/master/js/common/bridge.go#L47)
when a struct is exported in to goja runtime, the property names are converted to snake_cased if "js" tag is missing in the struct
ie: VUs -> v_us , VUsMax -> v_us_max

Reply:
Ah, I see, makes total sense.